digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/ManoeuvreService.scala

```diff
@@ -322,7 +321,15 @@ class ManoeuvreService(roadLinkService: RoadLinkService, eventBus: DigiroadEvent
     }
   }
 
-  private def createManoeuvreFromTrafficSign(trafficSignInfo: TrafficSignInfo): Seq[Long] = {
+  private def insertInaccurateAsset(trafficSignInfo: TrafficSignInfo, fromTrafficSignGenerator: Boolean, exception: Exception ): Seq[Long] = {
+    if (!fromTrafficSignGenerator )
+      throw exception
+
+    inaccurateDAO.createInaccurateAsset(trafficSignInfo.id, Manoeuvres.typeId, trafficSignInfo.roadLink.municipalityCode, trafficSignInfo.roadLink.administrativeClass )
```

Comment:
The asset Id to insert at inaccurate working list need to be the same type as the assetTypeId